# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/grub/gnu.property.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- grub-0.97/stage2/Makefile.am.vanilla	2021-05-09 17:18:52.943421272 +0200
+++ grub-0.97/stage2/Makefile.am	2021-05-09 17:19:18.179420266 +0200
@@ -278,4 +278,4 @@
 # General rule for making a raw binary.
 SUFFIXES = .exec
 .exec:
-	$(OBJCOPY) -R .note -R .comment -R .note.gnu.build-id -R .reginfo -R .rel.dyn -O binary $< $@
+	$(OBJCOPY) -R .note -R .comment -R .note.gnu.build-id -R .reginfo -R .rel.dyn -R .note.gnu.property -O binary $< $@
--- grub-0.97/stage1/Makefile.am.vanilla	2021-05-09 17:18:52.941421272 +0200
+++ grub-0.97/stage1/Makefile.am	2021-05-09 17:19:07.227420703 +0200
@@ -12,4 +12,4 @@
 
 SUFFIXES = .exec
 .exec:
-	$(OBJCOPY) -O binary $< $@
+	$(OBJCOPY) -O binary -R .note.gnu.property $< $@
